package.json

```diff
@@ -32,6 +32,9 @@
     "version": "ALLOW_MISSING_VARIABLES=1 yarn build"
   },
   "dependencies": {
+    "@chakra-ui/react": "^2.8.2",
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
```

Comment:
Why are we adding these extra dependencies?

Reply:
its a dependency for the chakra/ui

Reply:
I think we should use headless ui, because it doesn't require any peer deps except for react and react-dom, which is fair. It also requires a bit fewer dependencies in total and is considerably more popular. I would very much prefer not to have peer dependencies that are not explicitly used in the project.